tooling/camel-tooling-maven/src/main/java/org/apache/camel/tooling/maven/MavenDownloaderImpl.java

```diff
@@ -731,10 +731,7 @@ Settings mavenConfiguration(
         // 3) ${user.home}/.m2/repository (if exists)
         // 4) /tmp/.m2/repository
         String localRepository = System.getProperty("maven.repo.local");
-        if (localRepository == null || "".equals(localRepository.trim())) {
-            localRepository = settings.getLocalRepository();
```

Comment:
Why remove this?

Reply:
The first step. Another IF with the same condition below. Both get combined into one block.
The second step. The assigned value isn't used before another value is assigned to localRepository => the firas assignment can be removed

Reply:
Sorry I don't get it, what about the use case where `settings.getLocalRepository()` returns a non-empty value?

Reply:
My bad. I understand now